src/main/java/ru/sbt/mipt/oop/HallHandler.java

```diff
@@ -0,0 +1,37 @@
+package ru.sbt.mipt.oop;
+
+public class HallHandler implements Handler{
+    private final SmartHome home;
+
+    HallHandler(SmartHome home) {
+        this.home = home;
+    }
+
+    private void disableLights() {
+        for (Room room : home.getRooms()) {
+            for (Light light : room.getLights()) {
+                light.setOn(false);
+                SensorCommand command = new SensorCommand(CommandType.LIGHT_OFF, light.getId());
+                CommandSender commandSender = new CommandSender();
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D0%BF%D0%B5%D1%80%D0%B5%D0%B4%D0%B0%D0%B2%D0%B0%D0%B9%D1%82%D0%B5-%D0%B7%D0%B0%D0%B2%D0%B8%D1%81%D0%B8%D0%BC%D0%BE%D1%81%D1%82%D0%B8-%D1%81%D0%BD%D0%B0%D1%80%D1%83%D0%B6%D0%B8